tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -239,9 +250,10 @@ def test_dhcp_relay_after_link_flap(ptfhost, dut_dhcp_relay_data, validate_dut_r
         # Bring all uplink interfaces back up
         for iface in dhcp_relay['uplink_interfaces']:
             duthost.shell('ifconfig {} up'.format(iface))
-
-        pytest_assert(wait_until(50, 5, check_link_status, duthost, dhcp_relay['uplink_interfaces'], "up"),
-                      "Not all uplinks are up")
+            
```

Comment:
is the issue happening if the portchannel consists of one interface and doesn't happen otherwise?

Reply:
No, I don't think so. There are two members in each portchannel on this testbed
```
  No.  Team Dev         Protocol     Ports
-----  ---------------  -----------  ---------------------------
 0001  PortChannel0001  LACP(A)(Up)  Ethernet0(S) Ethernet4(S)
 0002  PortChannel0002  LACP(A)(Up)  Ethernet20(S) Ethernet16(S)
 0003  PortChannel0003  LACP(A)(Up)  Ethernet68(S) Ethernet64(S)
 0004  PortChannel0004  LACP(A)(Up)  Ethernet84(S) Ethernet80(S)
```